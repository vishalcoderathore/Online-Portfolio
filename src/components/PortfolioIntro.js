import React from 'react';

const PortfolioIntro = () => (
    <section className="py-5 text-center bg-light" id="aboutMe">
        <div className="container">
            <div className="row">
                <div className="col">
                    <div className="info-header">
                        <h1 className="intro--header pb-3 pt-5 ">
                            ABOUT ME
                    </h1>
                    </div>
                </div>
            </div>
            <div className="row">
                <div className="col-md-6 pt-5 px-2">
                    <p className="lead pb-3 intro--desc">
                        Hi! My name is Vishal Singh Rathore and I am a Software Developer located in Minneapolis, Minnesota. I've always been drawn to the overlap between design, develop and testing. My skills are broad: from front end to back end. I have worked with a variety of technologies including Java, PL/SQL, HTML, CSS, JavaScript and Jquery.
                </p>
                </div>
                <div className="col-md-6 pt-5 px-2">
                    <div className="container">
                        <div className="row">
                            <div className="col">

                                <p className="progress-title">HTML5/CSS3</p>
                                <div className="progress">
                                    <div className="progress-bar progressInfo1" >
                                    </div>
                                </div>

                                <p className="progress-title">JavaScript</p>
                                <div className="progress">
                                    <div className="progress-bar progressInfo2">
                                    </div>
                                </div>

                                <p className="progress-title">Java</p>
                                <div className="progress">
                                    <div className="progress-bar progressInfo3">
                                    </div>
                                </div>

                                <p className="progress-title">PL/SQL</p>
                                <div className="progress">
                                    <div className="progress-bar progressInfo4">
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
            <div className="row">
                <div className="col mb-5">
                    <img src="/images/vsr.jpg" alt="pic-not-found" className="img-fluid rounded-circle mb-3"></img>
                </div>
            </div>
        </div>
    </section>
);

export default PortfolioIntro;