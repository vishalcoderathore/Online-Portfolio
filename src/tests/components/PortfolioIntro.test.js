import React from 'react';
import PortfolioIntro from '../../components/PortfolioIntro';
import { shallow } from 'enzyme';

test('should render portfolio dashboard page correctly', () => {
    const wrapper = shallow(<PortfolioIntro />);
    expect(wrapper).toMatchSnapshot();
});