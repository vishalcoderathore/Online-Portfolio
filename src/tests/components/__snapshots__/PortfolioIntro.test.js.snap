// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render portfolio dashboard page correctly 1`] = `
<section
  className="py-5 text-center bg-light"
  id="home"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col"
      >
        <div
          className="info-header mb-5"
        >
          <h1
            className="intro--header pb-4"
          >
            ABOUT ME
          </h1>
        </div>
      </div>
    </div>
    <div
      className="row"
    >
      <div
        className="col-md-6 mb-5 px-2"
      >
        <p
          className="lead pb-3 intro--desc"
        >
          Hi! My name is Vishal Singh Rathore and I am a Software Developer located in Minneapolis, Minnesota. I've always been drawn to the overlap between design, develop and testing. My skills are broad: from front end to back end. I have worked with a variety of technologies including Java, PL/SQL, HTML, CSS, JavaScript and Jquery.
        </p>
      </div>
      <div
        className="col-md-6 mb-5 px-2"
      >
        <div
          className="container"
        >
          <div
            className="row"
          >
            <div
              className="col"
            >
              <p
                className="progress-title"
              >
                HTML5/CSS3
              </p>
              <div
                className="progress"
              >
                <div
                  className="progress-bar progressInfo1"
                />
              </div>
              <p
                className="progress-title"
              >
                JavaScript
              </p>
              <div
                className="progress"
              >
                <div
                  className="progress-bar progressInfo2"
                />
              </div>
              <p
                className="progress-title"
              >
                Java
              </p>
              <div
                className="progress"
              >
                <div
                  className="progress-bar progressInfo3"
                />
              </div>
              <p
                className="progress-title"
              >
                PL/SQL
              </p>
              <div
                className="progress"
              >
                <div
                  className="progress-bar progressInfo4"
                />
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="row"
    >
      <div
        className="col mb-5"
      >
        <img
          alt="pic-not-found"
          className="img-fluid rounded-circle mb-3"
          src="/images/vsr.jpg"
        />
      </div>
    </div>
  </div>
</section>
`;
